### DISCUSSION TITLE
Using curly brackets {} instead of colon for path parameters

### DISCUSSION THREAD

**Discussion by collinmue** [PaloAltoNetworks/docusaurus-openapi-docs#1049] [Q&A]
I'm using the plugin version 4.0.1  with Docusaurus 3.5.2. 

My instance generates the following path parameter style on the MDX pages:

`https://{baseURL}/accounts/:customerAccountId`

The YAML input looks like:

```
paths:
  /accounts/{customerAccountId}:
```

I would like to generate the mdx page with the same curly brackets around `{customerAccountId}`

What am I doing wrong?
Thank you very much for your help. 
PS: I've tried it with quotation marks in the YAML ("/accounts/{customerAccountId}":). That didn't help.

**Comment by sserrata** [comment]
Have you tried defining customerId as a server variable or path parameter?

**Reply by collinmue** [reply]
Thank you very much for your quick  response. 
Unfortunately, I had already defined the path parameter correctly under "components". After your suggestion, I tried moving the definition under the endpoint description. That didn't solve the issue either. 
The YAML file renders correctly in the Swagger Editor on the web.
